Units/Misc/v_ideCodeParser.pas

```diff
@@ -526,24 +526,24 @@ function TDeclaration.GetCleanText: string;
 var
   i: Integer;
   a: TDeclarationArray;
+  s: string;
 begin
-  Result := '';
-  if (fCleanText <> '') then
-    Result := fCleanText
-  else if (fStartPos <> fEndPos) and (fOrigin <> nil) then
+  if (fCleanText = '') and (fStartPos <> fEndPos) and (fOrigin <> nil) then
   begin
-    fCleanText := RawText;
+    s := RawText;
```

Comment:
These changes are to prevent race conditions. I think locking is overkill. In the rare case that two threads need this at the same time, just calculate it twice and store for future use.